### DISCUSSION TITLE
Menu "Data"

### DISCUSSION THREAD

**Discussion by Leeb64** [devemux86/cruiser#26] [Q&A] [answered]
It's about the third line in the Cruiser menu: "Data"
I mean that the term was chosen a bit unfortunate..?
I think "POI's" would be better."data" is pretty general.

**Comment by devemux86** [comment, marked as answer]
Thanks for the suggestion.

POI as an abbreviation is known in some languages, others have to translate the long "Points of Interest".
Also .poi files can contain a lot more than just points, so currently "Data" is used.

However, this feature is going to change in the future!
See here: https://github.com/devemux86/cruiser/discussions/11#discussioncomment-4571638
